src/query/src/main/scala/osmesa/query/model/Hashtag.scala

```diff
@@ -0,0 +1,21 @@
+package osmesa.query.model
+
+import io.circe.generic.JsonCodec
+
+import scala.util.Random
+
+
+@JsonCodec
+case class Hashtag(tag: String, count: Int)
```

Comment:
Should this be called "UserHashtag"? Seems specific to the user, as opposed to the hashtag for a campaign.